README.md

```diff
@@ -26,7 +26,27 @@ codebuild/bin/s2n_install_test_dependencies.sh
 codebuild/bin/s2n_codebuild.sh
 ```
 
-### Have a Question?
+## Quickstart for OSX (or other platforms)
+
+If you are building on OSX, or simply don't want to execute the entire build script above, you can use build tools like Ninja.
+
+### OSX
+
+An example of building on OSX:
+
+```sh
+brew install ninja cmake
+git clone https://github.com/${YOUR_GITHUB_ACCOUNT_NAME}/s2n.git
+mkdir s2n_build
+cd s2n_build
+
+# Build with debug symbols
+cmake -GNinja -DCMAKE_BUILD_TYPE=Debug ../s2n
```

Comment:
Should we have instructions on pointing to the OpenSSL install directory? For me I have to include -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/openssl@1.1/1.1.1g

Reply:
Added CMAKE_PREFIX_PATH